src/raw/requests.rs

```diff
@@ -100,12 +78,11 @@ impl KvRequest for RawBatchGet {
         &mut self,
         pd_client: Arc<PdC>,
     ) -> BoxStream<'static, Result<(Self::KeyData, Store<PdC::KvClient>)>> {
-        let mut keys = Vec::new();
-        mem::swap(&mut keys, &mut self.keys);
+        let keys = mem::replace(&mut self.keys, Default::default());
 
         pd_client
             .clone()
-            .group_keys_by_region(keys.into_iter())
+            .group_keys_by_region(keys.into_iter().map(Into::into))
```

Comment:
Why is this needed? Can we change `group_keys_by_region`so it is not necessary?

Reply:
I've tried and think we shouldn't. 
```
fn group_keys_by_region<K: AsRef<Key> + Send + Sync + 'static>(
        self: Arc<Self>,
        keys: impl Iterator<Item = K> + Send + Sync + 'static,
    )
```
If we introduce a second generic parameter (Into<K>), for most cases, the compiler will fail to  compute what type it is. However, if the type is `impl Into<Key>` here, it will succeed (because there will be no other possibility).